src/__test__/components/data-management/ShareExperimentModal.test.jsx

```diff
@@ -49,9 +49,8 @@ describe('Share expeirment modal', () => {
     await act(async () => render(
       <ShareExperimentModal
         onCancel={onCancel}
-        activeProject={{
-          experiments: [fake.EXPERIMENT_ID],
-          uuid: fake.PROJECT_ID,
+        experiment={{
+          id: fake.EXPERIMENT_ID,
           name: fake.PROJECT_NAME,
```

Comment:
`fake.EXPERIMENT_NAME`? maybe you can remove `fake.PROJECT_NAME` from `utils/constants`if we're not using it anymore.